src/Datadog.Trace.ClrProfiler.Native/packages.config

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="nlohmann.json" version="3.5.0" targetFramework="native" />
-  <package id="libddwaf" version="1.0.6" targetFramework="net462" />
+  <package id="libddwaf" version="1.0.7" targetFramework="net462" />
```

Comment:
Shouldn't this be `targetFramework="native"`?

Reply:
Yes, fixed in 702cc956